np.sh

```diff
@@ -18,8 +18,9 @@ fi
 trashCli=$(node -e "var path = require('path');console.log(path.join(path.dirname(require('fs').realpathSync('$0')), 'node_modules/.bin/trash'))");
 
 node "$trashCli" node_modules &&
-npm install &&
+npm install --production &&
 npm test &&
 npm version ${1:-patch} &&
 npm publish &&
-git push --follow-tags
+git push --follow-tags &&
+(npm install --silent --no-spin >/dev/null &) # get devDependencies back in the background
```

Comment:
I thinks this would be better: `npm install --silent --no-spin >/dev/null &!`. It explicitly disowns the process instead of doing it indirectly through being started in a subshell.